stress/backend/src/main/scala/kanaloa/stress/backend/MockBackend.scala

```diff
@@ -26,17 +38,23 @@ object MockBackend {
    * @param optimalThroughput maximum number of requests can handle per second
    */
   class BackendRouter(
+      throttle: Boolean,
       optimalConcurrency: Int,
       optimalThroughput: Int,
       bufferSize: Int = 10000,
+      minLatency: Option[FiniteDuration] = None,
       overloadPunishmentFactor: Option[Double] = None
   ) extends Actor with ActorLogging {
 
     var requestsHandling = 0
 
     val rand = new Random(System.currentTimeMillis())
     val perResponderRate = Math.round(optimalThroughput.toDouble / 10d / optimalConcurrency).toInt msgsPer 100.milliseconds
-    val baseLatency = perResponderRate.duration / perResponderRate.numberOfCalls
+
+    val baseLatency = {
+      val latencyFromThroughput = perResponderRate.duration / perResponderRate.numberOfCalls
+      if (minLatency.fold(false)(_ > latencyFromThroughput)) minLatency.get else latencyFromThroughput
```

Comment:
I see.  Yea..that's a bit hard to catch immediately.  Can't you just do a `collect` or a `filter` ?
`minLatency.filter( _ > latencyFromThroughput).getOrElse(latencyFromThroughput)` ?